src/contracts/middleware/Middleware.sol

```diff
@@ -162,20 +164,27 @@ contract Middleware is
     function stakeToPower(address vault, uint256 stake) public view override returns (uint256 power) {
         address collateral = vaultToCollateral(vault);
         string memory pairSymbol = collateralToPairSymbol(collateral);
-
+        console2.log("pairSymbol: ", pairSymbol);
+        console2.log("collateral: ", collateral);
         if (bytes(pairSymbol).length == 0) {
             revert Middleware__NotSupportedCollateral(collateral);
         }
 
         StorageMiddleware storage $ = _getMiddlewareStorage();
         (uint128 latestPrice, uint128 timestampOfLatestPrice) = IDIAOracleV2($.diaOracleAddress).getValue(pairSymbol);
-
+        console2.log("latestPrice: ", latestPrice);
+        console2.log("timestampOfLatestPrice: ", timestampOfLatestPrice);
         if (timestampOfLatestPrice == 0 || latestPrice == 0) {
             revert Middleware__InvalidData();
         }
 
         // Normalize power to 18 decimals, since the price is already on 8 decimals
-        power = stake.mulDiv(uint256(latestPrice), 10 ** 10);
+        power = stake.mulDiv(uint256(latestPrice), 10 ** 8);
```

Comment:
Maybe the comment is outdated? Is the price 10 now decimals? If so, where is this set? 
TBH I don't remember why it was 8 decimals here.